pkgs/tools/backup/zfs-replicate/default.nix

```
@@ -1,22 +1,28 @@
-{ buildPythonApplication, click, fetchPypi, hypothesis, pytest
-, lib, stringcase
+{ buildPythonApplication
+, click
+, fetchPypi
+, pytestCheckHook
+, hypothesis
+, pytest
+, lib
+, stringcase
 }:
 
 buildPythonApplication rec {
   pname = "zfs-replicate";
```

To match the fname on pypi:

```suggestion
  pname = "zfs_replicate";
```